### DISCUSSION TITLE
Generate Manifest file

### DISCUSSION THREAD

**Discussion by ntwaddell** [aws-samples/appstream-serverless-image-creation-automation#18] [General]
How do you generate a manifest file for an application?

**Comment by dbarlett** [comment]
Please see https://docs.aws.amazon.com/appstream2/latest/developerguide/customizing-appstream-images.html#optimizing-app-launch-performance (Windows) and https://docs.aws.amazon.com/appstream2/latest/developerguide/optimize-launch-performance.html (Linux).
